pkg/cloud/ibmcloud/deployer.go

```diff
@@ -89,3 +93,37 @@ func (d *DeploymentClient) GetKubeConfig(cluster *clusterv1.Cluster, master *clu
 	}
 	return strings.TrimSpace(parts[1]), nil
 }
+
+func getSSHKeyFile(homeDir string) string {
+	defaultKey := homeDir + "/.ssh/id_ibmcloud"
+	envPrivateKey, ok := os.LookupEnv("IBMCLOUD_HOST_SSH_PRIVATE_FILE")
+	if !ok || (0 == strings.Compare(envPrivateKey, "")) {
+		return defaultKey
+	}
+	klog.V(3).Infof("Found environment variable SSH private file %s ", envPrivateKey)
+	exist, _ := FileExists(envPrivateKey)
+	if exist {
+		return envPrivateKey
+	}
+	privateKey := filepath.Join(GetCurrentPath(), envPrivateKey)
+	exist, _ = FileExists(privateKey)
+	if !exist {
+		klog.Errorf("Cannot found SSH key file %s", privateKey)
+	}
+	return privateKey
+}
+
+func FileExists(file string) (bool, error) {
+	_, err := os.Stat(file)
+	if err == nil {
+		return true, nil
+	}
+	return false, err
+}
+
+func GetCurrentPath() string {
```

Comment:
please use `os.Getwd()`. `GetCurrentPath()` actually get path to the binary.

Reply:
Update code to get user current path but binary path,  user can `generate-yaml.sh` and `clusterctl` anywhere, such as put `clusterctl` under /usr/local/bin. :)